pkg/cmd/registry/create/create.go

```diff
@@ -194,9 +201,23 @@ func promptPayload(opts *options) (payload *srsmgmtv1.RegistryCreate, err error)
 		return nil, err
 	}
 
+	promptDescription := &survey.Input{
+		Message: opts.localizer.MustLocalize("registry.cmd.create.input.description.message"),
+		Help:    opts.localizer.MustLocalize("registry.cmd.create.input.description.help"),
+	}
+
+	err = survey.AskOne(promptDescription, &answers.Description)
```

Comment:
It is likely that a description is long, possibly multiline. We need to ensure that the user can create a multi-line description.